### PR TITLE
Fix Container Dropdown showing placeholder on page load

### PR DESCRIPTION
**Pull request by nicolethoen**
[CONSOLE-1226](https://jira.coreos.com/browse/CONSOLE-1226)

### CODE REVIEW

File: frontend/public/components/utils/dropdown.jsx [hunk 1]
```diff
@@ -447,7 +447,8 @@ export class ContainerDropdown extends React.PureComponent {
       items={dropdownItems}
       spacerBefore={spacerBefore}
       title={title}
-      onChange={onChange} />;
+      onChange={onChange}
+      noSelection={!_.isEmpty(currentKey)} />;
```
Comment (spadgett) [inline frontend/public/components/utils/dropdown.jsx]: This looks wrong... `noSelection` is meant to be used for action menus where an item isn't selected. In this dropdown, and item should be selected.

**Comment by spadgett** [comment]
We're hitting overview flakes in a few spots, @TheRealJon fyi

/retest

```
1) Visiting Overview page Deployments : displays a deployment in the project overview list
   Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
       at ontimeout timers.js:427:11
       at tryOnTimeout timers.js:289:5
       at listOnTimeout timers.js:252:5
       at Timer.processTimers timers.js:212:10
```

**Comment by TheRealJon** [comment]
@spadgett Yeah seems like every time I run the tests locally a different test case times out. Still looking into it.

**Pull request merged**